frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -328,117 +330,114 @@ const Map: React.FC<MapProps> = ({
   const [infoOpen, setInfoOpen] = React.useState(false);
   const [layersOpen, setLayersOpen] = React.useState(false);
   return (
-    <ReactResizeDetector handleWidth>
```

Comment:
this is just the removal of the reactresizedetector and addition of the resizeRef, no other changes.